packages/css-library/package.json

```diff
@@ -11,7 +11,7 @@
     "build:tokens": "style-dictionary build",
     "build:stylesheets": "sass --load-path=../../node_modules/@uswds/uswds/packages/ src/stylesheets:dist/",
     "build:minify": "yarn build:stylesheets --style compressed --no-source-map",
-    "build": "yarn run copy && yarn build:tokens && yarn build:stylesheets",
+    "build": "yarn run copy && yarn build:tokens && yarn build:stylesheets && yarn build:minify",
```

Comment:
I'm adding the minify step because it doesn't error so why not?

I think we'll still need to look into this issue though: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2295